### DISCUSSION TITLE
How to Get Data Rate in SX1280 in FLRC Mode

### DISCUSSION THREAD

**Discussion by Yash-Varuni** [jgromes/RadioLib#830] [Q&A]
### How to Get Bit Rate in SX1280 in FLRC Mode
I have Used SX1268, where I can use radio.getDataRate() to get the Data Transmit Rate, but when I use it for SX1280 FLRC Modem compile show error message error: 'class SX1280' has no member named 'getDataRate'.

**Comment by jgromes** [comment]
`getDataRate` returns the measured data rate and it is not implemented on SX1280. I'm actually thinking of removing it entirely, it's not really that useful.

**Reply by Yash-Varuni** [reply]
`getDataRate` Sometime its useful when we want to know at what rate data got transmitted.
